test/integration/matchers/expectHeading.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+exports.expectHeading = function(heading) {
```

Comment:
I don't, but I think this may be @benjaminapetersen overreacting to the fairly large helpers.js file.